addons/account/models/account_move.py

```diff
@@ -4336,8 +4336,8 @@ def create_tax_cash_basis_entry(self, percentage_before_rec):
                                 'account_id': account_id.id,
                                 'tax_exigible': True,
                                 'move_id': newly_created_move.id,
-                                'currency_id': line.currency_id.id,
-                                'amount_currency': self.amount_currency and line.currency_id.round(-line.amount_currency * amount / line.balance) or 0.0,
+                                'currency_id': self.currency_id.id,
+                                'amount_currency': self.amount_currency and self.currency_id.round(self.amount_currency * amount / line.balance) or 0.0,
```

Comment:
the sign `-` disappeared, is it intented?

Reply:
Yes it's intended as this created aml is a credit, the sign - must be removed as it is already included in the balance (debit - credit)

Reply:
`'amount_currency': line.currency_id.round(-line.amount_currency * amount / line.balance) if line.currency_id and line.balance else 0.0,
`